RecoEgamma/EgammaTools/plugins/DRNCorrectionProducerT.cc

```diff
@@ -405,20 +405,22 @@ void DRNCorrectionProducerT<T>::fillDescriptions(edm::ConfigurationDescriptions&
   TritonClient::fillPSetDescription(desc);
   desc.add<edm::InputTag>("particleSource");
   desc.add<edm::InputTag>("rhoName");
-  desc.add<edm::InputTag>("reducedEcalRecHitsEB");
-  desc.add<edm::InputTag>("reducedEcalRecHitsEE");
-  desc.add<edm::InputTag>("reducedEcalRecHitsES");
+  desc.add<edm::InputTag>("reducedEcalRecHitsEB", edm::InputTag("reducedEcalRecHitsEB"));
+  desc.add<edm::InputTag>("reducedEcalRecHitsEE", edm::InputTag("reducedEcalRecHitsEE"));
+  desc.add<edm::InputTag>("reducedEcalRecHitsES", edm::InputTag("reducedEcalRecHitsES"));
   descriptions.addWithDefaultLabel(desc);
 }
 
 //reco:: template instances are supported
 //uncomment the lines below to enable them
-//using PatPhotonDRNCorrectionProducer = DRNCorrectionProducerT<pat::Photon>;
+
+//using GsfElectronDRNCorrectionProducer = DRNCorrectionProducerT<reco::GsfElectron>;
 //using GedPhotonDRNCorrectionProducer = DRNCorrectionProducerT<reco::Photon>;
```

Comment:
Commented-out code is not preferred as it clutters the readability and obscures the purpose. If you think it's important to explicitly mention it, the single line `//reco:: template instances are supported` is sufficient IMO.

If the reco classes will be needed in the future, they can be trivially added, there's no need to write "potential code" just in case.